src/types/redis_set.cc

```diff
@@ -70,7 +70,11 @@ rocksdb::Status Set::Add(const Slice &user_key, const std::vector<Slice> &member
   WriteBatchLogData log_data(kRedisSet);
   batch->PutLogData(log_data.Encode());
   std::string sub_key;
+  std::unordered_set<std::string_view> mset;
   for (const auto &member : members) {
+    if (!mset.insert(member.ToString()).second) {
```

Comment:
better remove `ToString()`, `rocksdb::Slice` might has `ToStringView()`, or you can use `emplace({member.data(), member.size()})`